CHANGELOG.rst

```diff
@@ -2,6 +2,35 @@
 Change history
 ==============
 
+3.1.1 (2025-08-06)
```

Comment:
Since this adds a new feature (structlog), let's make it a minor release instead

```suggestion
3.2.0 (2025-08-07)
```